Week3/homework/7-step3.js

```diff
@@ -9,6 +9,12 @@ function f1(val) {
 f1(x);
 
 console.log(x);
+/* Here we have created a variable and assigned it to a value x 9. Then we created a function
+   and we pass a value to it, so the result should be 10, but here we just returned the 
+   value of constant x that has value 10. */
```

Comment:
The constant `x` has the value 9. We couldn't even change it to something else because we used `const` to declare it.